lib/fs/mtimefs.go

```diff
@@ -211,42 +215,44 @@ func (f mtimeFile) Stat() (FileInfo, error) {
 		return nil, err
 	}
 
-	real, virtual, err := f.fs.load(f.Name())
+	dbMtime, err := f.fs.Load(f.Name())
 	if err != nil {
 		return nil, err
 	}
-	if real == info.ModTime() {
+	if dbMtime.Real == info.ModTime() {
 		info = mtimeFileInfo{
 			FileInfo: info,
-			mtime:    virtual,
+			mtime:    dbMtime.Virtual,
 		}
 	}
 
 	return info, nil
 }
 
+// Used by copyRange to unwrap the real file to access the
 func (f mtimeFile) unwrap() File {
 	return f.File
 }
 
-// The dbMtime is our database representation
-
-type dbMtime struct {
-	real    time.Time
-	virtual time.Time
+// The DBMtime is our database representation
+type DBMtime struct {
```

Comment:
If we expose this we should give it a less crappy name, maybe something like `MtimeMapping` or so?